lib/beacon/blueprint_converter.ex

```diff
@@ -55,6 +55,9 @@ defmodule Beacon.BlueprintConverter do
   defp render_node(node) when is_binary(node), do: node
   # Special case to just output content as HTML
   defp render_node(%{"tag" => "raw", "content" => content}), do: content
+  defp render_node(%{"tag" => "img", "attributes" => attributes}) do
```

Comment:
This is unrelated but I found a bug with pages that contained `<img>` tags because those should be self-closed.